src/blog-utils/StaticAddonCard/index.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+import * as React from 'react';
+
+import { getAddonIconUrl } from 'amo/imageUtils';
+import { nl2br, sanitizeHTML } from 'amo/utils';
+import AddonBadges from 'amo/components/AddonBadges';
+import AddonTitle from 'amo/components/AddonTitle';
+import GetFirefoxButton, {
+  GET_FIREFOX_BUTTON_TYPE_ADDON,
+} from 'amo/components/GetFirefoxButton';
+import type { AddonType } from 'amo/types/addons';
+
+import './styles.scss';
+
+type Props = {|
+  addon: AddonType,
+|};
+
+const StaticAddonCard = ({ addon }: Props): React.Node => {
+  if (!addon) {
+    return null;
+  }
+
+  const summary = addon.summary ? addon.summary : addon.description;
+
+  return (
+    <div className="StaticAddonCard" data-addon-id={addon.id}>
+      <div className="AddonIcon">
+        <div className="AddonIconWrapper">
+          <img className="AddonIconImage" src={getAddonIconUrl(addon)} alt="" />
+        </div>
+      </div>
+
+      <AddonTitle addon={addon} />
+
+      <AddonBadges addon={addon} />
+
+      <p
+        className="AddonSummary"
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={sanitizeHTML(nl2br(summary), ['a', 'br'])}
```

Comment:
this is a deliberately more restrictive list of tags, I guess.  (the sanitized summary can have a wider number of tags)

Reply:
I thought the summary could only have those two html tags because it is a "autolink field"? I think we had an issue about that because Fenix wasn't allowing any html tags at all.

Note: this list of tags comes from AMO, I only copied it.

Reply:
Oh.  I looked at the list of tags for the underlying TranslatedField and it had a longer list, including the standard formatting ones, but it could easily have been overridden somewhere else.  Where did you get/copy the list from?

Reply:
This is the issue I mentioned above: https://github.com/mozilla/addons-server/issues/15669

The list and this code actually comes from: https://github.com/mozilla/addons-frontend/blob/cabfc342cdbd1fed6c9907051cdedceb8998971a/src/amo/pages/Addon/index.js#L427-L430

Reply:
you're right.   summary is a `LinkifiedTranslation` rather than `PurifiedTranslation`, and it does only support `a` tags (I guess `br` are added somewhere along the way from converting `\n` to `<br>`)